src/sentry/dynamic_sampling/tasks/utils.py

```diff
@@ -0,0 +1,21 @@
+from sentry.utils import metrics
+
+
+def dynamic_sampling_task():
+    """
```

Comment:
Also rather than doing:
```
def dynamic_sampling_task():
    def wrapper(func):
        def wrapped_func():
            ...
        return wrapper_func
    return wrapper
```

Why not do:
```
def dynamic_sampling_task():
    def wrapped_func():
        ...
    return wrapper_func
```
and then use:
`@dynamic_sampling_task` rather than `dynamic_sampling_task()`